### PR TITLE
Polish Ensembl Gene ID conversions (bonus reference updates!)

### PR DESCRIPTION
**Pull request by jashapiro**
### Purpose

Polishing Ensembl gene id conversions... 
Plus some reference updates

### Issue addressed

General content and formatting related to #290 and #289. These changes are primarily in `02-microarray/gene-id-annotation_microarray_01_ensembl.Rmd` and `03-rnaseq/gene-id-annotation_rnaseq_01_ensembl.Rmd`.

Rendered versions of these files can be viewed at: 

https://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/blob/jashapiro/polish-ensembl/02-microarray/gene-id-annotation_microarray_01_ensembl.html
and
https://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/blob/jashapiro/polish-ensembl/03-rnaseq/gene-id-annotation_rnaseq_01_ensembl.html

The biggest changes are a bit of extra explanation of where the `org.` packages come from, and their relationship to AnnotationDbi. Some of that was in the notebook, but not right where I thought the reader might want it. I also added explicit notes about where the `mapIds()` function comes from. 

Since I had to cite AnnotationDbi, I ended up getting a bit into citation updating, which makes this PR spread a bit wider than it probably ought to. See below about the trouble I caused myself.

I also removed all occurrences of `Refine.bio` and replaced them with `refine.bio`

#### Gotchas the reviewer should know about

I noticed that one of the references was labeled as `Igor2020`, but the author's last name is Dolgalev, so I updated it to `Dolgalev2020` and moved the reference. However, this will likely create conflicts with #434.

Similarly, I updated the AnnotationDbi reference to `Pages2020-package` (from `Carlson2020-package`) to reflect the current citation. 

I fully expect that I will have some conflicts to resolve before this merges, after #434 goes in.
This is a problem of my own making... I will be responsible for fixing it.

### CODE REVIEW

**Comment by jashapiro** [comment]
I am retargeting this to #434 (as a kind of stacked PR) to allow me to work on conflicts. I am assuming based on past experience than when #434 is merged it will retarget to staging. Also changing to draft so it doesn't accidentally get merged...

**Comment by jashapiro** [comment]
Fewer conflicts than expected! Still waiting in case others develop...

File: 02-microarray/gene-id-annotation_microarray_01_ensembl.Rmd [hunk 1]
```diff
@@ -283,27 +289,26 @@ head(mapped_df)
...
 
 We can see that our data frame has a new column `Symbol`.
-Let's get a summary of the gene symbols returned in the `Symbol` column of our mapped data frame.
+Let's get a summary of the gene symbols in the `Symbol` column of our mapped data frame.
 
 ```{r}
-# We can use the `summary()` function to get a better idea of the distribution of symbols in the `Symbol` column
-summary(mapped_df$Symbol)
+# Use the `summary()` function to show the distribution of Symbol values
+# We need to use `as.factor()` here to get the counts of unique values
```
Comment (cansavvy) [inline 02-microarray/gene-id-annotation_microarray_01_ensembl.Rmd:296]: Probably should add a short comment explaining the `maxsum` bit.

**Review by cansavvy — APPROVED** [review]
These all seem like good changes to me! 

> I noticed that one of the references was labeled as Igor2020, but the author's last name is Dolgalev, so I updated it to Dolgalev2020 and moved the reference. However, this will likely create conflicts with #434.
Similarly, I updated the AnnotationDbi reference to Pages2020-package (from Carlson2020-package) to reflect the current citation. 

I didn't check all the other notebooks that possibly use this reference, but is this something you search and replaced? This is the only thing I haven't checked up on.

File: 02-microarray/gene-id-annotation_microarray_01_ensembl.Rmd (hunk 1 above)
Comment (jashapiro) [reply 02-microarray/gene-id-annotation_microarray_01_ensembl.Rmd:296]: Done!

**Pull request merged**